neqo-http3/src/send_message.rs

```diff
@@ -177,7 +178,11 @@ impl SendStream for SendMessage {
         let available = conn
             .stream_avail_send_space(self.stream_id())
             .map_err(|e| Error::map_stream_send_errors(&e.into()))?;
-        if available <= 2 {
+        if available < MIN_DATA_FRAME_SIZE {
+            conn.stream_set_writable_event_low_watermark(
+                self.stream_id(),
+                NonZeroUsize::new(MIN_DATA_FRAME_SIZE).expect("MIN_DATA_FRAME_SIZE greater than 0"),
```

Comment:
> `NonZeroUsize::new` is const... But `Option::unwrap()` isn't...

I guess it will eventually https://github.com/rust-lang/rust/issues/67441.

Intuitively, I don't think a non-const `unwrap` has any performance hit here. If you think differently, I am happy to write a benchmark for it to find out.

Reply:
> I think that we can use `.unwrap()` rather than `.expect()` when the value we're unwrapping is set in a constant.

:+1: done in 2bc3e783.